mobile/src/tasks/obdTask.ts

```
@@ -0,0 +1,79 @@
+        await sleep(batch.length ? 60 : 150);
+      }
+    } catch (err) {
+      console.warn('[obdTask] error:', err);
+    } finally {
+      try { await obdService.stopPolling(); } catch {}
+      try { obdService.stopTrip(); } catch {}
+      try { locationService.stopLocationTracking(); } catch {}
+      try { if (await wakeLock.isHeld()) await wakeLock.release(); } catch {}
+      resolve();
+    }
+  });
```

Biome (2.1.2)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)